docs/source/conf.py

```diff
@@ -52,8 +51,8 @@
 d = {}
 try:
     version_path = os.path.join(base_path, '..', '..', 'traitsui', '_version.py')
-    with open(version_path, 'r', encoding='utf8') as fp:
-        exec(fp.read(), d)
+    with open(version_path, 'rb') as fp:
```

Comment:
Nitpick: reading a text file as bytes seems ... wrong. I think I prefer the original here, even with the ugliness of the extra  import.